Cargo.toml

```diff
@@ -19,10 +19,9 @@ default-target = "x86_64-pc-windows-msvc"
 targets = ["aarch64-pc-windows-msvc", "i686-pc-windows-msvc", "x86_64-pc-windows-msvc"]
 
 [dependencies]
-socket2 = "0.3.16"
 
 [dependencies.winapi]
-version = "0.3.3"
+version = "0.3.9"
```

Comment:
What's the reason for this upgrade? Users of `miow` will get the latest winapi automatically from `cargo update`. Is there some bugfix or functionality that we need/want here? Otherwise I don't see why we need to bump the *minimum* winapi version.